app/src/main/java/com/conupods/Settings/SettingsPersonalActivity.java

```diff
@@ -0,0 +1,56 @@
+package com.conupods.Settings;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.conupods.OutdoorMaps.View.MapsActivity;
+import com.conupods.R;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SettingsPersonalActivity extends AppCompatActivity {
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.settings_page_personal);
+
+        SharedPreferences preferences = getApplicationContext().getSharedPreferences("Preferences", Context.MODE_PRIVATE);
+        SharedPreferences.Editor prefEdit = preferences.edit();
+
+        Button done = findViewById(R.id.done1);
+        done.setOnClickListener(view -> {
+            startActivityIfNeeded(new Intent(SettingsPersonalActivity.this, MapsActivity.class).setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT), 0);
+        });
+
+        Button defaultPage = findViewById(R.id.toggle1_1);
+        defaultPage.setOnClickListener(view -> {
+            startActivityIfNeeded(new Intent(SettingsPersonalActivity.this, SettingsActivity.class).setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT), 0);
+        });
+
+        Button infoPage = findViewById(R.id.toggle1_2);
+        infoPage.setOnClickListener(view -> {
+            startActivityIfNeeded(new Intent(SettingsPersonalActivity.this, SettingsInfoActivity.class).setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT), 0);
+        });
+
+        Button myAccount = findViewById(R.id.myAccount);
+        myAccount.setOnClickListener(view -> {
+            //Implement account connection
+            //Default email
+            prefEdit.putString(String.valueOf(R.id.email), "defaultEmail@gmail.com").apply();
+            TextView email = findViewById(R.id.email);
+            email.setText(preferences.getString(String.valueOf(R.id.email), "No Email Found"));
+        });
```

Comment:
how would the user enter an email?

Reply:
good point, its better (at least) to pass a variable and assign it a dummy email for now, than passing the string directly.